internal/bundle/helpers.go

```diff
@@ -62,7 +54,7 @@ func initTestTemplate(t *testing.T, templateName string, config map[string]any)
 		return "", err
 	}
```

Comment:
If we reference this locally, there should no longer be a need to copy the template.